doc/doxygen/headers/dofs.h

```diff
@@ -84,6 +84,12 @@
   output     [label="Graphical output", URL="\ref output"];
   manifold   [label="Manifold",         URL="\ref manifold"];
 
+{rank=same
+  mapping -> quadrature        [dir="none", color="transparent"];
```

Comment:
Pretty unimportant, but we're all type A here: Can you indent these sub-graphs and put the `rank=same` on a line of its own?

Reply:
Better?